### PR TITLE
WIP Add flat secgroup and docs update for origin

### PR DESCRIPTION
**Pull request by bogdando**
Add an optional flat sec group (defaults to false). If enabled, trades some security for reduced number of sec rules, by merging master, node, etcd and infra rules. Helps to mitigate tenants' quotas.

Also add minor updates for docs and an example ansible.cfg.
WIP: Update required packages and docs for deploying OpenShift origin, f.e. on Centos7.

Rebased from master, thus contains unrelated changes.

### CODE REVIEW

File: playbooks/provisioning/openstack/README.md [hunk 1]
```diff
@@ -96,13 +105,13 @@ Assuming your OpenStack (Keystone) credentials are in the `keystonerc`
 file, this is how you stat the provisioning process:
 
     . keystonerc
-    ansible-playbook -i inventory  --private-key ~/.ssh/openshift openshift-ansible-contrib/playbooks/provisioning/openstack/provision.yaml
+    ansible-playbook -i inventory --timeout 30  --private-key ~/.ssh/openshift openshift-ansible-contrib/playbooks/provisioning/openstack/provision.yaml
```
Comment (bogdando) [inline playbooks/provisioning/openstack/README.md]: note, for my case, the "Timeout (Xs) waiting for privilege escalation prompt:" error was poping up for 15 and 30s timeout. Worked well with --timeout 90 tho

Comment (tomassedovic) [inline playbooks/provisioning/openstack/README.md]: Yeah I wonder why it sometimes takes so long.

**Comment by tomassedovic** [comment]
Thanks! This looks great and I'm happy to merge it, but: what's with the first two commits by cooktheryan & pshiffe? They seem to be about AWS and GCE and not related to this.

Can you remove them from this PR, please?

**Comment by tomassedovic** [comment]
I've merged it manually, removing the extra commits. Closing this PR. Thanks!

**Pull request closed without merging**